ruma-events/src/room/message.rs

```diff
@@ -59,6 +65,57 @@ pub enum MessageEventContent {
     Video(VideoMessageEventContent),
 }
 
+/// Enum modeling the different ways relationships can be expressed in a
+/// `m.relates_to` field of an m.room.message event.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(from = "RelatesToJsonRepr")]
+#[serde(into = "RelatesToJsonRepr")]
+pub enum Relation {
+    /// A reference to another event.
+    Reference(Reference),
```

Comment:
Just to make sure it's clear what I meant before:

```suggestion
    #[cfg(feature = "unstable-pre-spec")]
    Reference(Reference),
```